Sources/Subs-Db-mysqli.php

```diff
@@ -238,6 +238,13 @@ function smf_db_replacement__callback($matches)
 				smf_db_error_backtrace('Wrong value type sent to the database. Date expected. (' . $matches[2] . ')', '', E_USER_ERROR, __FILE__, __LINE__);
 		break;
 
+		case 'time':
+			if (preg_match('~^([0-1]?\d|2[0-3]):([0-5]\d):([0-5]\d)$~', $replacement, $time_matches) === 1)
+				return sprintf('\'%02d:%02d:%02d\'', $time_matches[1], $time_matches[2], $time_matches[3]);
```

Comment:
STR_TO_DATE is the similiar function like pg : https://dev.mysql.com/doc/refman/5.5/en/date-and-time-functions.html#function_str-to-date
pattern find you there: https://dev.mysql.com/doc/refman/5.5/en/date-and-time-functions.html#function_date-format